### DISCUSSION TITLE
Do leptos have IndexedDb?

### DISCUSSION THREAD

**Discussion by Raduc4** [leptos-rs/leptos#649] [Ideas]
I have been trying to use IndexedDb with another framework, but unfortunately, no luck.
Will Leptos consider adding this feature to the framework in the near future?

Thank you.

**Comment by gbj** [comment]
You can use IndexedDb with Leptos via the ordinary `web-sys` bindings for it. It's out of scope for the framework to provide anything additional here, I think.  But you can find some extensive notes on using Leptos with IndexedDb [here](https://underscorefunk.github.io/build-apps-with-rust/client/store_data/indexeddb.html)

**Comment by underscorefunk** [comment]
Hey there @Raduc4 . I'm the person who was writing the doc referenced by @gbj . You can totally use IndexedDb with Leptos. The article is not complete yet and the book I'm writing about leptos is still in a very rough draft form. IndexedDb requires a lot of explanation and I do plan to get back to it. I hope that you find the article useful, even in it's incomplete state. :)

**Reply by Raduc4** [reply]
Thank you a lot.
